Sources/Charcoal/Filters/Root/RootFilterViewController.swift

```diff
@@ -474,44 +469,45 @@ private extension RootFilterViewController {
         guard let freeTextFilter = filterContainer.freeTextFilter else { return }
 
         if freeTextFilterViewController == nil {
-            freeTextFilterViewController = FreeTextFilterViewController(filter: freeTextFilter, selectionStore: selectionStore)
-            freeTextFilterViewController?.delegate = self
-            freeTextFilterViewController?.filterDelegate = freeTextFilterDelegate
-            freeTextFilterViewController?.filterDataSource = freeTextFilterDataSource
-        }
+            let freeTextFilterViewController = FreeTextFilterViewController(filter: freeTextFilter, selectionStore: selectionStore)
+            self.freeTextFilterViewController = freeTextFilterViewController
+            freeTextFilterViewController.delegate = self
+            freeTextFilterViewController.filterDelegate = freeTextFilterDelegate
+            freeTextFilterViewController.filterDataSource = freeTextFilterDataSource
 
-        let searchBar = freeTextFilterViewController!.searchBar
-        tableView.addSubview(searchBar)
+            tableView.addSubview(freeTextFilterViewController.searchBar)
```

Comment:
Was it an intentional change to add `searchBar` as subview only on 1st call of this function? It is `.removeFromSuperview()` up there, so might cause trouble if this func is supposed to add it back? 🤔